pkg/exporter/syslogexporter/exporter.go

```diff
@@ -43,27 +42,9 @@ func initExporter(cfg *Config, createSettings exporter.CreateSettings) (*sysloge
 	var tlsConfig *tls.Config
 	var err error
 
-	if cfg.CACertificate != "" {
-		var serverCert []byte
-		serverCert, err = os.ReadFile(cfg.CACertificate)
-		if err != nil {
-			return nil, err
-		}
-
-		pool := x509.NewCertPool()
-		pool.AppendCertsFromPEM(serverCert)
-		tlsConfig = &tls.Config{
-			RootCAs: pool,
-		}
-	}
-
-	if cfg.Certificate != "" && cfg.Key != "" {
-		var certificate tls.Certificate
-		certificate, err = tls.LoadX509KeyPair(cfg.Certificate, cfg.Key)
-		if err != nil {
-			return nil, err
-		}
-		tlsConfig.Certificates = []tls.Certificate{certificate}
+	tlsConfig, err = cfg.TLSSetting.LoadTLSConfig()
```

Comment:
will this make adding all cert options mandatory?

Reply:
No, it is as earlier, it creates tls config from configuration https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/configtls.go